### PR TITLE
Laborator 6 - ALU

### PR DESCRIPTION
**Pull request by VictorCaproiu**

### CODE REVIEW

File: chapters/microprogramable_cpu/arithmetic-logic-unit/drills/README.md [hunk 1]
```diff
@@ -34,7 +34,50 @@ localparam SHR  = 4'd8;
...
 ```
 
-TODO: exaplain the remaining code
+## Operațiile
+
+Partea combinațională a modulului descrie comportamentul fiecărei operații în funcție de codul operațional ``` i_w_opcode ``` prin intermediul unei structuri `case`.
+
+### Adunarea cu carry (ADC)
+
+Se execută adunarea, iar în cazul în care rezultatul depășește că mărime parametrul `p_data_width`, bitul în plus activează flag-ul _carry_.
```
Comment (DTeodor-Alexaandru) [inline chapters/microprogramable_cpu/arithmetic-logic-unit/drills/README.md:43]:
```suggestion
Se execută adunarea celor 2 operanzi și carry, iar în cazul în care rezultatul depășește ca mărime parametrul `p_data_width`, bitul în plus activează flag-ul _carry_.
```

**Review by DTeodor-Alexaandru — CHANGES_REQUESTED** [review]
Is this PR tested?

**Comment by VictorCaproiu** [comment]
Yes, works fine on the nexys board.

**Review by DTeodor-Alexaandru — CHANGES_REQUESTED** [review]
Please center the images and then the PR is ready to be merged

File: chapters/microprogramable_cpu/arithmetic-logic-unit/reading/README.md [hunk 2]
```diff
@@ -2,8 +2,12 @@
 
 Unitatea aritmetică-logică este responsabilă de efectuarea operațiilor aritmetice și logice în timpul execuției instrucțiunilor. Operațiile primesc unul sau doi operanzi, iar UAL-ul în afară de producerea rezultatului setează și o serie de indicatori de condiții (eng. *flags*) rezultați în urma operațiilor. Operațiile disponibile în UAL derivă din instrucțiunile prezente în setul de instrucțiuni al procesorului didactic, însă nu au neapărat o corespondență 1-la-1 cu acestea. Unele operații sunt folosite în mai multe instrucțiuni, iar unele instrucțiuni folosesc mai multe operații. UAL-ul trebuie însă proiectat în așa fel astfel încât să cuprindă toate operațiile necesare în execuția instrucțiunilor disponibile în procesorul didactic.
 
+<div align="center">
+
 ![Unitatea aritmetică-logică](../media/fig_ual.png)
 
+</div>
```
Comment (DTeodor-Alexaandru) [inline chapters/microprogramable_cpu/arithmetic-logic-unit/reading/README.md:9]: Move this after Figure

**Review by DTeodor-Alexaandru — CHANGES_REQUESTED** [review]

**Review by DTeodor-Alexaandru — APPROVED** [review]

**Pull request merged**